Duplicati/Library/Modules/Builtin/SendMail.cs

```diff
@@ -231,42 +231,10 @@ protected override void SendMessage(string subject, string body)
             List<string> servers = null;
             if (string.IsNullOrEmpty(m_server))
             {
-                var dnslite = new DnsLib.DnsLite();
-                var dnslist = new List<string>();
+                var dnsclient = new LookupClient();
+                var records = dnsclient.Query(toMailDomain, QueryType.MX).Answers.MxRecords();
 
-                //Grab all IPv4 addresses
-                foreach (NetworkInterface networkInterface in NetworkInterface.GetAllNetworkInterfaces())
-                    try
-                    {
-                        foreach (IPAddress dnsAddress in networkInterface.GetIPProperties().DnsAddresses)
-                            if (dnsAddress.AddressFamily == AddressFamily.InterNetwork)
-                                dnslist.Add(dnsAddress.ToString());
-                    }
-                    catch (Exception ex) { Logging.Log.WriteExplicitMessage(LOGTAG, "DNSServerLookupFailure", ex, "Failed to get DNS servers from network interface"); }
-
-                dnslist = dnslist.Distinct().ToList();
-
-                // If we have no DNS servers, try Google and OpenDNS
-                if (dnslist.Count == 0)
-                {
-                    // https://developers.google.com/speed/public-dns/
-                    dnslist.Add("8.8.8.8");
-                    dnslist.Add("8.8.4.4");
-
-                    //http://www.opendns.com/opendns-ip-addresses/
-                    dnslist.Add("208.67.222.222");
-                    dnslist.Add("208.67.220.220");

```

Comment:
I was initially concerned about the removal of these DNS servers, but it looks like the `LookupClient` will fall back to the Google DNS servers if no local DNS servers are found.

https://dnsclient.michaco.net/docs/DnsClient.NameServer.html#DnsClient_NameServer_ResolveNameServers_System_Boolean_System_Boolean_